tests/integration/framework/process/grpc/operator/operator.go

```diff
@@ -156,8 +156,8 @@ func New(t *testing.T, fopts ...Option) *Operator {
 	o.GRPC = procgrpc.New(t, append(opts.grpcopts,
 		procgrpc.WithServerOption(func(t *testing.T, ctx context.Context) grpc.ServerOption {
 			secProv, err := security.New(ctx, security.Options{
-				SentryAddress:           "localhost:" + strconv.Itoa(opts.sentry.Port()),
-				ControlPlaneTrustDomain: "localhost",
+				SentryAddress:           "127.0.0.1:" + strconv.Itoa(opts.sentry.Port()),
```

Comment:
This will break on IPv6-only environments

Reply:
For testing, it is OK if this fixes it. If not, I revert.